.travis.yml

```diff
@@ -1,4 +1,12 @@
 language: node_js
 node_js:
-  - "7"
+  - "8"
+  - "node"
 cache: npm
+script:
+  - npm run test-all
+  - npm run check-security
+  - npm run lint
+branches:
+  only:
+    - master
```

Comment:
i think you want to run travis on all branches since travis should be required for PR

Reply:
yes, we do it. The config restricts building branches but does not stop building pull-requests.

May seem overkill now as the project is too small but took advantage of this PR to make it. fyi browser-laptop shares [a similar config](https://github.com/brave/browser-laptop/blob/master/.travis.yml#L17).